bin/install-solr.sh

```diff
@@ -35,6 +37,16 @@ create_collection() {
     echo "Created collection $name"
 }
 
+create_schema() {
```

Comment:
Please can we update these CURL scripts to specify which fields we are indexing and which we are not? Thanks Richard.

Also please can we ensure that the schema is effectively locked. I think we can do this by disabling automatic field guessing. Details are listed at [this link](https://lucene.apache.org/solr/guide/8_1/schemaless-mode.html)

Reply:
Ahh, forgot to upload the changes. Done now